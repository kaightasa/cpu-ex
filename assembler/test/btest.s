branch:
    add r3, r1, r2
    sub r5, r3, r4
calc:
    cmpwi r3, 10
    beq calc