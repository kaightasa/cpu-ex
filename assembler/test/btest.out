c611000
10a32000
5803000a
3c000001
